docs/introduction/scope.md

```diff
@@ -1,30 +1,29 @@
 # Scope
 
-Gekko is a free and open source tool that is designed as a starterskit for automated trading on cryptocurrency markets. Gekko aims to have a low barrier entry to writing your own strategies (however a basic scripting knowledge in javascript is required for users who create their own strategies). This page will discuss the scope of some components of Gekko, explaining how things are done and why they are done a certain way.
+Gekko is a free and open source starter kit for automated trading on cryptocurrency markets. Gekko has a low barrier to entry as only minimal knowledge of javascript is required to write trading strategies.
 
 ## Market data
 
-Gekko aggregates all market data into minutely candles (OHLC, VWP and amount of trades). This means Gekko only has to store candles on disk, which take up a predictable amount of space on your harddrive. The tradebot will use some additional market data (the orderbook) to make efficient execute orders efficiently, but this data is not visibile anywhere else.
+Gekko aggregates market data into [OHLC candles](https://en.wikipedia.org/wiki/Open-high-low-close_chart). Gekko only stores candle data on disk taking up a predictable amount of hard drive space. The tradebot will use some additional market data (Such as the orderbook) in order to efficiently execute orders, but this data cached in memory and not accessible.
 
 ## Strategies
 
-Strategies are simple scripts that handle new market data (OHLC candles) as well as calculated indicator results (strategies specify what indicators they want with which settings). Every time there is new data the strategy can determine to signal either LONG or SHORT. That's about it! This very simple design (candles + indicator values go in => signals come out) is quite powerfull.
+Strategies are scripts that receive market data in the form of (OHLC candles) and execute buy / sell orders which could involve the calculation and usage of trading indicators. See [Creating a Strategy](../strategies/creating_a_strategy.html) for more information.
 
-Unfortunately this simple design can sometimes be limiting, here are some limitations:
+The limits of this design are::
 
-- Since strategies are only fed candles (and indicator values):
- - Strategies cannot act on a smaller timeframe than 1 minute.
- - Strategies cannot see any trades.
- - Strategies cannot look at the orderbook.
+- The smallest time frame a strategy can act on is one minute.
+- Strategies do not see trades
+- Strategies cannot look at the order book.
 - Strategies do not know what the current portfolio looks like.
-- Strategies can only trigger a LONG or SHORT which signals to go "all in".
+- Strategies can either trigger a `LONG` (Sell all shares) or `SHORT` (Buy up to the current budget allowance).
 
-## Execution strategy
+## Strategy Execution
```

Comment:
This is definitely not strategy execution, this is the execution strategy: they are 2 different concepts and execution in this one refers to order execution.